Master.py

```diff
@@ -61,6 +61,9 @@ def initialization():
     param["n_low"] = (Ind_all_low[:, 1] - Ind_all_low[:, 3] + 1).astype(int)[0]  # number of columns
     param["GeoRef"] = calc_geotiff(Crd_all, res_desired)
     timecheck('End')
+    # Display Intial Information
+    print('\nRegion: ' + param["region"] + ' - Year: ' + str(param["year"]))
+    print('Folder Path: ' + paths["region"] + '\n')
     return paths, param
```

Comment:
It would be good to display information about region name, year, and the folder in which the results can be found, other information might be added here... What do you think ?